src/dc_tools.rs

```diff
@@ -889,7 +892,7 @@ fn as_path_unicode<'a>(s: *const libc::c_char) -> &'a std::path::Path {
 pub(crate) fn time() -> i64 {
     SystemTime::now()
         .duration_since(SystemTime::UNIX_EPOCH)
-        .unwrap()
+        .unwrap_or_default()
```

Comment:
This can only fail when the system time is set before the unix epoch.  That's.... i think crashing is fair enough at that point because who knows what else would break in this case.